packages/flutter/test/widget/input_test.dart

```diff
@@ -633,7 +633,11 @@ void main() {
     TestGesture gesture = await tester.startGesture(firstPos, pointer: 7);
     await tester.pump();
     await gesture.moveBy(new Offset(0.0, -1000.0));
-    await tester.pump();
+    await tester.pump(const Duration(seconds: 2));
+    // Wait and drag again to trigger https://github.com/flutter/flutter/issues/6329
+    // (No idea why this is necessary, but the bug wouldn't repro without it.)
```

Comment:
Just file a bug describing how to trigger something manually, and point to this code saying that for some reason the test framework needs this to make it trigger. So long as the issue is clear enough about how to reproduce the difference, eventually I'll get around to investigating it.